services/conda/conda-version.service.js

```diff
@@ -6,28 +7,28 @@ export default class CondaVersion extends BaseCondaService {
   static category = 'version'
   static route = { base: 'conda', pattern: ':variant(v|vn)/:channel/:pkg' }
 
-  static examples = [
-    {
-      title: 'Conda',
-      namedParams: { channel: 'conda-forge', package: 'python' },
-      pattern: 'v/:channel/:package',
-      staticPreview: this.render({
-        variant: 'v',
-        channel: 'conda-forge',
-        version: '3.7.1',
-      }),
+  static openApi = {
+    '/conda/{variant}/{channel}/{package}': {
```

Comment:
I'd completely forgotten I had done this because I actually worked on this PR months ago, but I think this was my logic:

In javascript, `package` is not a legal variable name in strict mode (which ESM implies) https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#future_reserved_words

```
var package = 'foo'
    ^^^^^^^

SyntaxError: Unexpected strict mode reserved word
```

so we don't call route params `package` because they become variable names. We generally use `pkg` or `packageName`. I think in this case, I noticed that the openApi params don't actually have to match the route as long as the positions match up, so I think I decided to present this as "package" in the frontend.

Maybe the best thing to here is have them all match up but change them both to `packageName`?

Reply:
> Maybe the best thing to here is have them all match up but change them both to packageName?

Yes, let's do that!